wee_slack.py

```diff
@@ -369,7 +369,7 @@ def buffer_input_cb(b, buffer, data):
     channel = channels.find(buffer)
     if not channel:
         return w.WEECHAT_RC_OK_EAT
-    reaction = re.match("(\d*)(\+|-):(.*):", data)
+    reaction = re.match("^\s*(\d*)(\+|-):(.*):\s*$", data)
```

Comment:
because of the anchors, they would have to type something on a line by itself to match. if they put any non whitespace character before _or_ after, the regex won't match.

i.e.

"+:blah:" = Match
"    +:blah:"  = Match
"2+:blah:   " = Match

but

"2+:blah: isn't working for me!" = No match
"I'm trying to do +:blah:" = No match
" +:blah:??" = No match

It seems like someone typing a reaction with just leading space as a way to show the text of the reaction would be rare.